docs/blog/2019-01-29-themes-update-child-theming-and-component-shadowing/index.md

```diff
@@ -0,0 +1,131 @@
+> Gatsby
+> Themes](https://www.gatsbyjs.org/blog/2018-11-11-introducing-gatsby-themes/)
+> for a written introduction and [my Gatsby Days
+> talk](https://www.youtube.com/watch?v=wX84vXBpMR8) for a more audio/visual
+> approach.
+
+# Child Theming
+
+We've merged [a PR](https://github.com/gatsbyjs/gatsby/pull/10787) into Gatsby
+core to support Child theming. Child theming is an extension of the core theming
+algorithm to support a "parent/child" style relationship where child themes can
+rely on parent themes in the same way sites rely on themes. This means you can
+now use the `__experimentalThemes` gatsby-config key in a theme as well as your
+sites.
+
+This change is being made to make it easier for theme authors to produce themes
+that rely on complex behavior while enabling users who have different skill sets
+to consume and modify those themes. Theme authors will benefit from the ability
+to abstract logic, UI elements, and other Gatsby idioms into multiple packages
+which can be consumed as a single theme. Child themes (and themes as a whole)
+will improve the ability of people with different skill sets to create
+performant, beautiful sites. A few that I had in mind while building out child
+theming are:
+
+- A designer who wants to change a set of design tokens to create a whole new
+  look for a theme
+- Someone who is more familiar with HTML and CSS than React and GraphQL wants to
+  create a set of child blog themes
+- Someone familiar with GraphQL wants to enable new functionality by writing
+  their own queries for page templates
+
+I also want child themes to enable an ecosystem to build on top of and compose
+with each other. The 80% use case for a blog should be capable of being built on
+a similar data model encapsulated in a parent theme so that users can swap out
+child themes to try out new sites or new looks. This will take some more effort
+to achieve, so more on that at a later date.
+
+## A Concrete Example
+
+Taking advantage of child theming requires some familiarity with theming itself
+to be used effectively. If you're already experienced with themes and want to
+see how one possible way to upgrade your themes to use this feature, the
+corresponding PR in the [gatsby-theme-examples
+repo](https://github.com/ChristopherBiscardi/gatsby-theme-examples) has [also
+been
+merged](https://github.com/ChristopherBiscardi/gatsby-theme-examples/pull/13).
+
+Let's say you have a theme that contains a blog data model and some logic
+relating to how to render blog posts at specific pages, we'll call it
+`gatsby-theme-blog-core`. This theme renders JSON representations of blog posts
+at the relevant page locations instead of full UI because it's only meant to
+encapsulate the data model. We'll skip going over page creation logic in
+`gatsby-node.js` and instead focus on the following possible `gatsby-config.js`
+for the core theme.
+
+```js
+module.exports = {
+  plugins: [`gatsby-mdx`],
```

Comment:
Is this the correct plugin or code example?

Reply:
yeah, it could be anything. Would something feel better to you here?

Reply:
I wasn't sure if it was the right example, since it seemed somewhat unrelated to the topic. Maybe the formatted titles thing I mentioned in a separate comment would help?